Formula/execline.rb

```diff
@@ -17,7 +17,6 @@ class Execline < Formula
   def install
     system "./configure",
       "--prefix=#{prefix}",
-      "--disable-static",
```

Comment:
So: you can delete the existing formulae and install the non-static parts (e.g. binaries) into `libexec` of the `s6` cellar.